test-suite/non-functional-tests/test-scenario-1/start-docker-test.sh

```diff
@@ -0,0 +1,16 @@
+#!/bin/bash
+
+openssl pkcs12 -export -out jmeterkeystore.p12 -inkey ../certs/client.key -in ../certs/client.crt -passout pass:password
+
+echo Stopping docker container:
+docker ps -a -q -f name=nf-test-1 | xargs -r docker stop
+echo Removing Docker container:
+docker ps -a -q -f name=nf-test-1 | xargs -r docker rm
```

Comment:
Would `docker compose rm -s -f` be simpler?

Reply:
yep, tested that and that works so have updated to this